agent/src/modules/webview.ts

```diff
@@ -25,7 +25,7 @@ async function get(handle: string): Promise<ObjC.Object> {
   for (const kind of WebViewKinds) {
     const clazz = ObjC.classes[`${kind}WebView`]
     try {
-      return getInstance(clazz, handle)
+      return await getInstance(clazz, handle)
```

Comment:
this is redundant

Reply:
actually it isn't. Without awaiting over the result, i wasn't able to get the URL of a WKWebView.
That's because it is first searching for UIWebView instances, fails and does not continue searching WKWebView instances.